docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -264,23 +257,23 @@ subtasks which are invoked in sequence:
 Once all of these commands have completed there are a couple of final tasks left that
 aren't strictly able to be automated:
 
-* If this is a stable release, it will be necessary to perform a post-release merge
+* It will be necessary to perform a post-release merge
   on open source. This normally will require you to merge the temporary release branch into the
   source branch (e.g. merge 3.2.4 into 3.2), or sometimes create new branches if
   releasing a new minor version, and bumping up the branch-alias in composer.json.
   E.g. branching 3.3 from 3, and aliasing 3 as 3.4.x-dev. You can then delete
   the temporary release branches. This will need to be done before updating the 
   release documentation in stage 3.
 * Merging up the changes in this release to newer branches, following the 
-  SemVer pattern (e.g. 3.2.4 > 3.2 > 3.3 > 3 > master). The more often this is
+  SemVer pattern (e.g. 4.2.4 > 4.2 > 4 > master). The more often this is
   done the easier it is, but this can sometimes be left for when you have
   more free time. Branches not receiving regular stable versions anymore (e.g.
-  3.0 or 3.1) should usually be omitted.
+  3.0 or 3.1) can be omitted.
 * Set the github milestones to completed, and create placeholders for the next
   minor versions. It may be necessary to re-assign any issues assigned to the prior
   milestones to these new ones.
 * Make sure that the [releases page](https://github.com/silverstripe/silverstripe-installer/releases)
-  on github shows the new tag. 
+  on github shows the new tag. [what to do if it's not showing?!]
```

Comment:
It's all well and good saying that the release should show, but what should we do if it doesn't?

Reply:
I guess it's be a case of manually going into the module, checking the tag exists and pushing it. If not, re-run the release process? I assume it would do these checks again and re-tag if necessary